corehq/form_processor/serializers.py

```diff
@@ -53,6 +54,25 @@ class Meta:
         exclude = ('id',)
 
 
+class HumanReadableChoiceField(serializers.ChoiceField):
+    def get_attribute(self, obj):
+        choice = super(serializers.ChoiceField, self).get_attribute(obj)
+        return self.choices[choice]
```

Comment:
Could do something like this if it's hard to support multiple choices: https://github.com/dimagi/commcare-hq/blob/master/corehq/form_processor/models.py#L1057-L1057